libraries/core_libs/consensus/include/pbft/pbft_chain.hpp

```diff
@@ -22,39 +22,115 @@ class DagBlock;
 struct Transaction;
 
 /**
- * In pbft_chain, two kinds of blocks:
- * 1. PivotBlock: determine DAG block in pivot chain
- * 2. ScheduleBlock: determine sequential/concurrent set
+ * @brief PbftChain class is a single linked list that contains finalized PBFT blocks
  */
 class PbftChain {
  public:
   explicit PbftChain(blk_hash_t const& dag_genesis_hash, addr_t node_addr, std::shared_ptr<DbStorage> db);
 
+  /**
+   * @brief Get PBFT chain head hash
+   * @return PBFT chain head hash
+   */
   blk_hash_t getHeadHash() const;
+
+  /**
+   * @brief Get PBFT chain size
+   * @return PBFT chain size
+   */
   uint64_t getPbftChainSize() const;
+
+  /**
+   * @brief Get PBFT chain size excluding empty PBFT blocks
+   * @return PBFT chain size excluding empty PBFT blocks
+   */
   uint64_t getPbftChainSizeExcludingEmptyPbftBlocks() const;
+
+  /**
+   * @brief Get last PBFT block hash
+   * @return last PBFT block hash
+   */
   blk_hash_t getLastPbftBlockHash() const;
 
+  /**
+   * @brief Get a PBFT block in chain
+   * @param pbft_block_hash PBFT block hash
+   * @return PBFT block
+   */
   PbftBlock getPbftBlockInChain(blk_hash_t const& pbft_block_hash);
+
+  /**
+   * @brief Get a unverified PBFT block
+   * @param pbft_block_hash PBFT block hash
+   * @return a unverified PBFT block
+   */
   std::shared_ptr<PbftBlock> getUnverifiedPbftBlock(blk_hash_t const& pbft_block_hash);
+
+  /**
+   * @brief Get a bunch of PBFT blocks in string
+   * @param period start period
+   * @param count number of PBFT blocks
+   * @param hash if return PBFT blocks hashes only
+   * @return a bunch of PBFT blocks in string
+   */
   std::vector<std::string> getPbftBlocksStr(size_t period, size_t count, bool hash) const;  // Remove
```

Comment:
please add this is only used by single RPC call

Reply:
Fixed